typescript/src/fork.ts

```diff
@@ -0,0 +1,40 @@
+import { createBucketFork } from "@tigrisdata/storage";
```

Comment:
Meta comment: 

In the examples, it is better to show one way of doing it, which is passing the `bucket` name as a top level parameter. Many ways of doing it is confusing.